<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>620</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>img.tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>791</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="load_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>100</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="convert_btn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resize_btn">
         <property name="text">
          <string>Resize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optimize_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Optimize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="crop_btn">
         <property name="text">
          <string>Crop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cc_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Color Correction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="blur_btn">
         <property name="text">
          <string>Blur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="censore_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Censore</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
